src/Promise.res

```diff
@@ -103,5 +101,11 @@ let catch = (promise, callback) => {
   })
 }
 
+external unsafeFromRejectable: rejectable<'a> => t<'a> = "%identity"
+
+let fromRejectable = (promise, successCb, errCb) => {
+  promise->unsafeFromRejectable->then(successCb)->catch(errCb)
```

Comment:
This could/should be done in a single `promise.then(successCb, errCb)`.

Of course it would need a binding for the 2-parameter version of `then`. (This could be added as `then2` in the .res file and not exposed in the .resi).

Reply:
No it can't just be an external, because `catch` requires specific wrapper logic to unify our `exn` value

Reply:
Ah I see, sorry!